lizmap/modules/lizmap/install/upgrade_presentation.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class lizmapModuleUpgrader_userfields extends jInstallerModule
```

Comment:
you should use the new classes of Jelix 1.8 `\Jelix\Installer\Module\Installer` (install.php should use it too, but it can be done into an other patch)